rewards/calc_rewards.py

```diff
@@ -87,17 +95,19 @@ def propose_root(
     chain: str,
     start: int,
     end: int,
-    past_rewards,
+    past_rewards: Dict,
     tree_manager: TreeManager,
     save=False,
-):
+) -> None:
     """
     Propose a root on a chain
 
     :param chain: chain to propose root
     :param start: start block for rewards
     :param end: end block for rewards
     :param save: flag to save rewards file locally, defaults to False
```

Comment:
save duplicated here

Reply:
Not really, one entry is describing param, enother one is describing the type of that param